src/vellum/workflows/runner/runner.py

```diff
@@ -605,9 +605,15 @@ def _handle_work_item_event(self, event: WorkflowEvent) -> Optional[NodeExecutio
             return event
```

Comment:
revert all changes to workflow runner for now. I think we could solve this with only changes to `src/vellum/workflows/nodes/displayable/bases/base_prompt_node/node.py`